src/webgl/interaction.js

```diff
@@ -88,20 +88,22 @@ p5.prototype.orbitControl = function(sensitivityX, sensitivityY, sensitivityZ) {
     this._setProperty('wheelDefaultDisabled', true);
   }
 
-  const scaleFactor = this.height < this.width ? this.height : this.width;
-
+  const scaleFactor = 50;
```

Comment:
Do you think we can make this scale factor be relative to the width and height? It's also used to orbit via click and drag, and when the canvas is much larger than 50, it means dragging across the canvas rotates more than one would expect. Here's an example of the current behaviour, and then after a quick change to try to make it something relative to the canvas size:

![zoomscalefactor](https://user-images.githubusercontent.com/5315059/211696599-4fd461e8-9882-4e39-8115-9ec14e12cedd.gif)